packages/tree_hash/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "tree_hash"
+version = "0.8.0"
+edition = "2021"
+description = "Efficient Merkle-hashing as used in Ethereum consensus"
+license = "Apache-2.0"
```

Comment:
Use this workspace

Reply:
Agreed. It was just a quick fork I did last night to make cosmwasm builds work

Reply:
Actually, I want to talk about how we do want to do this. Do we want to keep this fork here in this repo or keep it separate? Ideally, I want to find a better solution to this problem, which I have outlined in a separate issue: https://github.com/cosmos/solidity-ibc-eureka/issues/155

Reply:
Yes, agreed. If we just keep it here for now, then I agree and I will change that :)

Reply:
fixed